README.md

```diff
@@ -4,15 +4,15 @@
 
 This brief discussion paper aims to start a debate on whether a percentile is the best single delay metric and, if so, which percentile the industry should converge on.
 
 Note that the question addressed here is how to characterize a varying delay metric. That is orthogonal to the questions of what delay to measure and where to measure it. For instance, whether delay is measured in the application, at the transport layer or just at the bottleneck queue depends on the topic of interest and is an orthogonal question to the focus of this paper. Similarly, for delay under load, the question of which background traffic pattern to use depends on the scenario of interest and is an orthogonal question to the focus of this paper; which is solely about how to characterize delay variability most succinctly and usefully in *any* of these cases.
 
 ## Don't we need two metrics?
 
-In systems that aim for a certain delay, it has been common to quote mean delay and jitter. The distribution of delay is usually asymmetric, mostly clustered around the lower end, but with a long tail of higher delays. A traditional jitter metric is insensitive to the shape of this tail, because it is dominated by the *average* variability in the bulk of the traffic around the mean. However, it doesn't matter how little or how much variability there is in all the traffic that arrives before the play-out time. It only matters how much traffic arrives too late. The size of all the lower-than-average delay should not be allowed to counterbalance a long tail of above-average delay. 
+In systems that aim for a certain delay, it has been common to quote mean delay and jitter. The distribution of delay is usually asymmetric, mostly clustered around the lower end, but with a long tail of higher delays. Many jitter metrics are insensitive to the shape of this tail, because they are dominated by the *average* variability in the bulk of the traffic around the mean. [FYI, in our SCTE paper we quoted 9 different definitions of jitter in common usage in the industry] However, it doesn't matter how little or how much variability there is in all the traffic that arrives before the play-out time. It only matters how much traffic arrives too late. The size of all the lower-than-average delay should not be allowed to counterbalance a long tail of above-average delay. 
```

Comment:
Yes, ok to cite the SCTE paper. 

I guess I exaggerated when I said there were 9 different metrics :), but in any case there are 5 different ones.

Not sure I agree that most (or many) of the metrics "are dominated by the *average* variability in the bulk of the traffic around the mean."  Two of them probably are (the IPDV metric and the std.dev(PDV) metric), but the other three (P99 PDV, max(PDV), RMS(PDV)) are not.  My guess is that std.dev or IPDV are used more commonly than the others, so perhaps we could say that in predominant usage, your statement is true.

FYI, we ran one sequence of packet latency measurements from an ns3 application scenario through all 5 of those jitter definitions, and got the results that ranged from 0.06ms with one metric to 142.6ms with another.   So, I would rather say something that equates to "jitter is largely meaningless, unless a precise definition is given".

Reply:
Surely RMS(PDV) "measures the average variability in the bulk of traffic", much like std.dev(PDV). So 3 out of 5 measure average variability. Of the other 2, MLab NDT uses max(PDV), which is "obviously" unpredictable (I guess it gave your 142.6ms result). And Sam Knows uses P99, 'cos obviously Sam Knows best :)

So strictly, 'most' is not incorrect, and I wouldn't describe 3 as 'many'. I would rather not say "most commonly used", because SamKnows is pretty widespread.

Also I would rather not say anything about how disparate the jitter metrics are, 'cos that's off-topic, even if it is your pet peeve.

So what are we going to say? We must ensure that no-one reviewing or reading it feels it is insulting their approach. I think we have given a technical enough argument against focusing on variability in the cluster. So pls suggest any changes, otherwise I will leave it as is.